examples/web-form-with-express/public/client.js

```diff
@@ -0,0 +1,136 @@
+// This file is run by the browser each time your view template is loaded
+
+/**
+ * Define variables that reference elements included in /views/index.html:
+ */
+
+// Forms
+const dbForm = document.getElementById("databaseForm")
+const pageForm = document.getElementById("pageForm")
+const blocksForm = document.getElementById("blocksForm")
+const commentForm = document.getElementById("commentForm")
+// Table cells where API responses will be appended
+const dbResponse = document.getElementById("dbResponse")
+const pageResponse = document.getElementById("pageResponse")
+const blocksResponse = document.getElementById("blocksResponse")
+const commentResponse = document.getElementById("commentResponse")
+
+/**
+ * Functions to handle appending new content to /views/index.html
+ */
+
+// Appends the API response to the UI
+const appendApiResponse = function (res, el) {
+  console.log(res)
+
+  const result = document.createElement("p")
```

Comment:
optional, nit: it was a bit hard to grok this code because of variables `res` and `result`, separately. Could `result` be renamed to something like `newHtmlElement`, `newParagraph`, `resultElement`, or something along those lines?

if you make this change, would be great to do that in functions below as well

Reply:
good point, yes! Ty!